pkg/apis/workflow/v1alpha1/container_set_template_types.go

```diff
@@ -0,0 +1,70 @@
+package v1alpha1
+
+import corev1 "k8s.io/api/core/v1"
+
+type ContainerSetTemplate struct {
+	Graph        *ContainerGraph      `json:"graph,omitempty" protobuf:"bytes,1,rep,name=graph"`
+	Sequence     *ContainerSequence   `json:"sequence,omitempty" protobuf:"bytes,2,rep,name=sequence"`
+	VolumeMounts []corev1.VolumeMount `json:"volumeMounts,omitempty" protobuf:"bytes,3,rep,name=volumeMounts"`
+}
```

Comment:
@alexec I thought this morning you said we were going with the approach of removing `graph` and `sequence` and not having a sequence at all. In other words your second preference:

> My 2nd preference would be to have containers and eschew sequenced containers completely. Users would have to use dependencies/depends forever. The syntax and implementation will be simpler: both more user friendly and less buggy.

Reply:
My bad. It might be useful if I actually push my change _before_ asking for a review. The changes are now push.